tests/CMakeLists.txt

```diff
@@ -198,12 +198,16 @@ function(add_cts_test)
   # To make check that any .cpp files are passed
   # List created because, direct check on "${ARGN}" gives false-positive result
   set(tests_list "${ARGN}")
+  get_filename_component(test_dir ${CMAKE_CURRENT_SOURCE_DIR} NAME)
   if (tests_list)
-    get_filename_component(test_dir ${CMAKE_CURRENT_SOURCE_DIR} NAME)
     set(test_exe_name ${test_dir})
```

Comment:
Then this `test_dir` variable is now undefined?

Reply:
Hm, why, am I missing something? Now we just defining it in advance before `if` block, so we don't have to do it both in `if` and `else` blocks.

Reply:
Ah yes. I do not know why I have not seen the change above too!
Sorry for the confusion. :frowning_face: